bridge_adaptivity/bridge_lti/provider.py

```diff
@@ -102,6 +103,11 @@ def learner_flow(request, lti_consumer, params, collection_id=None):
         lti_consumer=lti_consumer,
         defaults={'course_id': params['context_id']}
     )
+    if created:
```

Comment:
I would rewrite in the string formatting statement: `log.debug("LTI user {}: user_id='{}'".format('created' if created else 'picked', lti_user.user_id))`